stdlib/public/Concurrency/TaskCancellation.swift

```diff
@@ -61,25 +61,16 @@ extension Task {
      }
   }
 
-  /// Returns `true` if the task is cancelled, and should stop executing.
+  /// A Boolean value that indicates whether the task should stop executing.
```

Comment:
I feel a bit uneasy dropping the "is cancelled" even though it reads like "cancelled is cancelled" i guess...

I somehow would like to indicate that all callers of isCancelled on this task, i.e. static funcs `Task.isCancelled` running within the task also will notice this.

Reply:
I was rewriting to avoid the content-free abstract problem of just repeating in English what the symbols in code already said.  Does the fact about all callers noticing cancellation belong here, or in the higher level discussion of cancellation at the top in that subsection?  Or maybe adding something here like:

> This value is used by other APIs, like the static `Task.isCancelled` property.